<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTrainInfoTable</class>
 <widget class="QDialog" name="CTrainInfoTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimuTutor 培训信息</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTableWidget" name="listTrainInfo">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;宋体&quot;;</string>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>用户ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>用户名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>主机ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>主机名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>任务分发ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>任务ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>任务名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>工况ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>工况名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>故障ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>故障名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>登录状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>运行状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>仿真速度</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>默认画面</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>任务状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>工况状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>故障状态</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="buttonTrainInfoAdd">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>添加培训信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonTrainInfoDel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>删除培训信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonTrainInfoModify">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>修改培训信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonExport">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>导出为EXCEL</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonRefresh">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>刷  新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonExit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>退  出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listTrainInfo</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonExit</sender>
   <signal>clicked()</signal>
   <receiver>CTrainInfoTable</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>705</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
